placement/options.go

```diff
@@ -60,15 +59,6 @@ func (o options) SetLooseRackCheck(looseRackCheck bool) Options {
 	return o
 }
 
-func (o options) AcrossZones() bool {
```

Comment:
Discussed offline, this is and likely will never used and has become a maintenance burden so fine to remove